content/source/docs/cloud/api/organizations.html.md

```diff
@@ -275,72 +360,104 @@ curl \
-      "can-start-trial": false,
-      "can-update-agent-pools": false
-    },
-    "fair-run-queuing-enabled": true,
-    "saml-enabled": false,
-    "owners-team-saml-role-id": null,
-    "two-factor-conformant": true
-  },
-  "relationships": {
-    "oauth-tokens": {
-      "links": {
-        "related": "/api/v2/organizations/hashicorp/oauth-tokens"
-      }
-    },
-    "authentication-token": {
-      "links": {
-        "related": "/api/v2/organizations/hashicorp/authentication-token"
-      }
+  "data": {
+    "id": "hashicorp",
+    "type": "organizations",
+    "attributes": {
+      "external-id": "org-Bzyc2JuegvVLAibn",
+      "created-at": "2021-08-30T18:09:57.561Z",
+      "email": "user@examplecom",
+      "session-timeout": null,
+      "session-remember": null,
+      "collaborator-auth-policy": "password",
+      "plan-expired": false,
+      "plan-expires-at": null,
+      "plan-is-trial": false,
+      "plan-is-enterprise": false,
+      "cost-estimation-enabled": false,
+      "name": "hashicorp",
+      "permissions": {
+        "can-update": true,
+        "can-destroy": true,
+        "can-access-via-teams": true,
+        "can-create-module": true,
+        "can-create-team": false,
+        "can-create-workspace": true,
+        "can-manage-users": true,
+        "can-manage-subscription": true,
+        "can-manage-sso": false,
+        "can-update-oauth": true,
+        "can-update-sentinel": false,
+        "can-update-ssh-keys": true,
+        "can-update-api-token": true,
+        "can-traverse": true,
+        "can-start-trial": true,
+        "can-update-agent-pools": false,
+        "can-manage-tags": true,
+        "can-manage-public-modules": true,
+        "can-manage-public-providers": false,
+        "can-create-provider": false
+      },
+      "fair-run-queuing-enabled": true,
+      "saml-enabled": false,
+      "owners-team-saml-role-id": null,
+      "two-factor-conformant": false
     },
-    "entitlement-set": {
-      "data": {
-        "id": "org-MxtxBC6ihhU6u8AG",
-        "type": "entitlement-sets"
+    "relationships": {
+      "oauth-tokens": {
+        "links": {
+          "related": "/api/v2/organizations/hashicorp/oauth-tokens"
+        }
       },
-      "links": {
-        "related": "/api/v2/organizations/hashicorp/entitlement-set"
+      "authentication-token": {
+        "links": {
+          "related": "/api/v2/organizations/hashicorp/authentication-token"
+        }
+      },
+      "entitlement-set": {
+        "data": {
+          "id": "org-Bzyc2JuegvVLAibn",
+          "type": "entitlement-sets"
+        },
+        "links": {
+          "related": "/api/v2/organizations/hashicorp/entitlement-set"
+        }
+      },
+      "subscription": {
+        "links": {
+          "related": "/api/v2/organizations/hashicorp/subscription"
+        }
       }
     },
-    "subscription": {
+    "links": {
+      "self": "/api/v2/organizations/hashicorp"
+    }
+  },
+  "included": [
```

Comment:
is this really in there? i think this would require `?include=entitlement_set` in the request URL. can either remove this or add it to the url above. i'd vote for removing this.

separately, do we note the included relations available for each endpoint? it seems like a thing we should do. the code is a bit tough to track down but it is explicitly defined for each resource controller.

Reply:
So, I hit the endpoint myself and created a test organization and this is the output - I just ran it again and got the same thing with the entitlement set included. Are you able to test as well - It could be that I did something wrong, but the new organization shows up on our test account just fine! 

As for the things that each object can include, we do list them at the bottom of the page: https://www.terraform.io/docs/cloud/api/workspaces.html#available-related-resources. I've seen this done a few times. But it doesn't look like we do it here for organizations - good catch! Would you be willing to point me in the right direction to get those and/or provide that content?

Reply:
In your test, was `?include=entitlement_set` in the URL? That is indeed what the web app uses to load organizations but for the API itself that's not default behavior. So I think it should be left out here.

It looks like `entitlement_set` is the only [allowable include](https://github.com/hashicorp/atlas/blob/1c0e74a114befbfe403db322e14efc9830958897/app/controllers/api/v2/organizations_controller.rb#L140-L142) ("Available Related Resources") for organizations. It looks like we don't call that out here https://www.terraform.io/docs/cloud/api/organizations.html#available-related-resources. But I'd guess that billing details aren't all that useful for customers in this context.

Reply:
TLDR we tested and this IS included! So I'm going to resolve :-)